plugin.rb

```diff
@@ -52,6 +52,15 @@ def translate
       raise Discourse::InvalidParameters.new(:post_id) if post.blank?
       guardian.ensure_can_see!(post)
 
+      if !user_group_allowed?
+        raise Discourse::InvalidAccess.new(
+                "not_in_group",
+                SiteSetting.restrict_translation_by_group,
+                custom_message: "not_in_group.title_translation",
```

Comment:
Does the `"not_in_group.title_translation"` string exist?

Reply:
I didn't knew exactly how to put it, since I saw in Discourse core code this is the way it is used, I thought It could be added later, let me know if this is not the right way. I tried checking the current implementation in Discourse core but maybe that is not the way to go since this is a plugin